pr_review_queue.py

```diff
@@ -397,8 +415,7 @@ def create_pr_review_queue(pull_request_list):
                 reviewers = ', '.join(get_slack_userid(requested_reviewer['login']) for requested_reviewer in pull_request['requested_reviewers'])
                 needs_review.append(f"{entry} {reviewers}")
             # 4. Needs conflict resolution or rebasing
-            elif (not pull_request['changes_requested'] and
-                  (pull_request['mergeable_state'] in {'dirty', 'behind'})):
+            elif (pull_request['mergeable_state'] in {'dirty', 'behind'}):
```

Comment:
I understand and agree with the motivation for this change, but won't this lead to PRs showing up twice on the list?